.mk/verify.mk

```diff
@@ -50,31 +50,45 @@ tidy:
 addlicense: $(TOOLBIN)/addlicense
 	$(ABSTOOLBIN)/addlicense -ignorefile .addlicenseignore -f $(ROOT_DIR)/hack/boilerplate.txt .
 
-GOLANGCI_LINT_FLAGS := \
+GOLINT_LINTERS ?= \
+	--disable-all \
+	--enable=deadcode \
 	--enable=dogsled \
 	--enable=dupl \
+	--enable=errcheck \
 	--enable=gocritic \
 	--enable=gofmt \
 	--enable=golint \
+	--enable=gosimple \
 	--enable=govet \
+	--enable=ineffassign \
 	--enable=misspell \
 	--enable=nakedret \
-	--enable=staticcheck \
+	--enable=structcheck \
+	--enable=typecheck \
 	--enable=unconvert \
+	--enable=unused \
+	--enable=varcheck \
 	--enable=whitespace
 
-#CODE_MAINT += lint
+GOLINT_VERBOSE ?= \
+	--verbose \
+	--print-resources-usage
+
+GOLINT_GOGC ?= 20
+
+CODE_MAINT += lint
 .PHONY: lint
 lint: $(TOOLBIN)/golangci-lint
-	$(TOOLBIN)/golangci-lint run $(GOLANGCI_LINT_FLAGS) ./...
+	GOGC=${GOLINT_GOGC} $(TOOLBIN)/golangci-lint run ${GOLINT_VERBOSE} ${GOLINT_LINTERS} ./...
```

Comment:
- Can GOLINT_VERBOSE be removed now?
- Can GOLINT_GOGC be removed now?

Reply:
yes.
yes.